src/main/kotlin/lotto/domain/Lotto.kt

```diff
@@ -0,0 +1,41 @@
+package lotto.domain
+
+class Lotto(lottoNumber: List<Int>) {
+    private val _lottoNumber = lottoNumber
+    val lottoNumber
+        get() = _lottoNumber
```

Comment:
```suggestion
class Lotto(val lottoNumber: List<Int>) {
```
Lotto가 불변 컬렉션이 되면서 backing property 없이 구현해도 될 것 같습니다.
[방어적 복사](https://tecoble.techcourse.co.kr/post/2021-04-26-defensive-copy-vs-unmodifiable/)를 생각하신 거라면 아래와 같이 구현할 수도 있겠네요 :)

```kotlin
class Lotto(lottoNumber: List<Int>) {
    private val lottoNumber = lottoNumber.toList()
```